libcontainer/configs/spec.go

```diff
@@ -32,6 +35,13 @@ func ParseSpec(s *specs.Spec) (*Config, error) {
 		}
 	}
 
+	// Setting default memory to pass to runnc as an argument.
+	if s.Linux.Resources.Memory.Limit != nil {
+		memory = utils.ConvertBytes(s.Linux.Resources.Memory.Limit)
```

Comment:
I see no reason for this to call a function. Do the division inline:
```
memory = (*s.Linux.Resources.Memory.Limit) / (1<<20)
```
. . . and maybe add a comment if it seems unclear to you